datamol/conformers/_conformers.py

```diff
@@ -33,6 +33,9 @@ def generate(
     warning_not_converged: int = 10,
     random_seed: int = 19,
     add_hs: bool = True,
+    fallback_to_random_coords: bool = True,
+    ignore_failure: bool = False,
+    custom_embed_params: dict = None,
     verbose: bool = False,
```

Comment:
I would replace `custom_embed_params` by `embed_params`. It's shorter and `custom_` seems unneeded for the meaning of the arg.